### PR TITLE
NR4 release blog post

### PR DESCRIPTION
**Pull request by knolleary**
Blog post to coincide with the NR4 release, highlighting the features of value to the FF audience.

*Note* do not merge without my explicit approval as it has to coordinate with the release and availability on FF Cloud.

### CODE REVIEW

File: src/blog/2024/06/node-red-4-on-flowfuse-cloud.md
Comment (Hasmin-AC) [inline src/blog/2024/06/node-red-4-on-flowfuse-cloud.md:3]:
```suggestion
subtitle: Making use of the Multiplayer Mode to collaborate with your teams
```

File: src/blog/2024/06/node-red-4-on-flowfuse-cloud.md
Comment (Hasmin-AC) [inline src/blog/2024/06/node-red-4-on-flowfuse-cloud.md:10]:
```suggestion
   - Node-RED
   - FlowFuse Cloud
```

File: src/blog/2024/06/node-red-4-on-flowfuse-cloud.md
Comment (Hasmin-AC) [inline src/blog/2024/06/node-red-4-on-flowfuse-cloud.md:15]:
```suggestion
Node-RED 4.0 is the new major release of the project, which is at the heart of all we do at FlowFuse. It brings a range of new features that continue to make Node-RED the first choice for low-code development.
```

**Comment by knolleary** [comment]
Marking ready for review as the release is on track for today. But still - do not merge without my approval.

**Comment by knolleary** [comment]
Ready for this to be merged as soon as its approved.

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**
